build/configs/rtl8730e/loadable_ext_ddr/defconfig

```diff
@@ -165,7 +165,7 @@ CONFIG_RTL8730E_UART1=y
 CONFIG_RTL8730E_UART4=y
 CONFIG_RTL8730E_SERIAL_FIFO=y
 CONFIG_AMEBASMART_MIPI=y
-CONFIG_AMEBASMART_USBDEVICE=y
+#CONFIG_AMEBASMART_USBDEVICE=y
```

Comment:
Please change to
#CONFIG_AMEBASMART_USBDEVICE is not set

Hi @sunghan-chang ,
I suggest that if the USB-related features are no longer supported or actively used, we should create a PR to disable the CONFIG_AMEBASMART_USBDEVICE setting by default for all defconfig.

Reply:
Yes. I disabled this temporarily for test. I will revert this change.